ui/src/App.vue

```diff
@@ -1,9 +1,12 @@
 <template>
-  <div class="container">
-    <SearchBar @termSubmit="onSubmit"></SearchBar>
-    <div class="row">
-      <CardListItem></CardListItem>
-    </div>
+  <div class="ui container">
+    <h2 class="ui center aligned header">Job Hooroom</h2>
+    <!-- <SearchBar @termSubmit="onSubmit"></SearchBar> -->
+      <SearchBar></SearchBar>
+      <CardListItem :locations="locations"></CardListItem>
+      <CardListItem :employmentTypes="employmentTypes"></CardListItem>
+      <CardListItem :skills="skills"></CardListItem>
+      <CardListItem :isRemote="isRemote"></CardListItem>
```

Comment:
Should loop over dynamic data using `v-for`: https://vuejs.org/v2/guide/list.html 
instead of specifying each element manually